airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -60,11 +66,15 @@ def get_conn(self):
             self.log.debug("loading kube_config from: default file")
             config.load_kube_config()
         else:
-            with tempfile.NamedTemporaryFile() as temp_config:
-                self.log.debug("loading kube_config from: connection kube_config")
-                temp_config.write(extras.get("extra__kubernetes__kube_config").encode())
-                temp_config.flush()
-                config.load_kube_config(temp_config.name)
+            path_or_payload = extras.get("extra__kubernetes__kube_config")
```

Comment:
I would prefer it to be a separate option to avoid difficult-to-debug situations.  If the file does not exist, the user should see a message that the file does not exist.

Reply:
Got it, I will separate it then